examples/network_connectivity_center/main.tf

```diff
@@ -15,7 +15,10 @@
  */
 
 module "network_connectivity_center" {
-  source       = "terraform-google-modules/network/google//modules/network-connectivity-center"
+  # source  = "terraform-google-modules/network/google//modules/network-connectivity-center"
+  # version = "~> 12.0"
+  source = "../../modules/network-connectivity-center"
```

Comment:
We should use the registry in examples:
```
  # source  = "terraform-google-modules/network/google//modules/network-connectivity-center"
  # version = "~> 12.0"
```

Module swapper will automatically point to the local version for the test and lint.

Reply:
sorry I forgot to remove it after local testing. Fixed